docs/user-manual/onboarding-guide/programguide

```diff
@@ -0,0 +1,112 @@
+# Deployment Guide
+
+**CiviForm Business Owner Deployment Guide**  
+This document is for someone that has decided to use CiviForm and now needs a guide of how to deploy it in their organization. It includes an overview as well as a detailed guide of what steps are needed at  each stage. It will help you:
+
+* Choose your initial programs  
+* Create test and production forms for programs  
+* [Optimize the program administration process / program mapping](https://docs.civiform.us/user-manual/onboarding-guide/journey-mapping)  
+* Simplify forms and the application process  
+* Set up user testing for forms / program applications  
+* Prepare for your launch  
+* Join the CiviForm community
+
+---
+
+## Overview
+
+CiviForm is a powerful open source tool for civic entities and governments designed to simplify the administration of benefit programs, and make it easier for residents to access those benefits. It lets residents and community organizations find and apply to multiple services in one place. CiviForm also makes it easier for governments to administer their programs so they can better reach communities in need. CiviForm deployments will be most successful if:
+
+* Your CiviForm deployment has an owner and champion.  
+* You can decide early as to which will be the first program(s) you will deploy on CiviForm.  
+* You can simplify and optimize your program applications.  
+* You can simplify how you process and follow up on applications.
+
+To ensure a successful CiviForm implementation, read through the brief guide below and create your plan to build programs.
+
+## Understanding Your Needs and Goals
+
+- [ ] **Define user needs:** To create program forms that people will find intuitive and easy to use, consider the needs of both your residents and program administrators.  
+      - [ ] Spend time to understand how your residents interact with the program applications so that you can optimize the experience of CiviForm users (including residents and administrators):  
+            - [ ] Review [Preparing for Human-Centered Redesign](https://beeckcenter.georgetown.edu/report/preparing-for-human-centered-redesign/).  
+            - [ ] Read more about [mapping the applicant experience](https://github.com/usds/benefits-enrollment-prototype/blob/master/assets/discovery-findings-mapping-enrollment-Nov2016.pdf).  
+      - [ ] Consider engaging community-based organizations for user research.
+
+- [ ] **Decide who will be the [CiviForm Admin](https://docs.civiform.us/overview/how-does-civiform-work\#civiform-admin-experience)** across your civic entity  
+      - [ ] Decide who will own and maintain your CiviForm instance in the long term (IT team, Digital Services or other). This will be the person who creates the Forms and Programs on CiviForm.
```

Comment:
**question:** Should this be a sub-bullet? Seems a bit unrelated to the parent.

Reply:
Thanks. I can't see which line you are referencing. Can you make a comment in the Google Doc for now?